### ISSUE TITLE
Cannot build on Windows after 593bde7

### ISSUE THREAD

**Issue by lennondotw** [ntop/n2n#164]
The latest commit 1b19003 on the dev branch **could not** be built successfully. But I remember that I was able to build successfully a few days ago. So I tested the commits these days and I found:

- 593bde7 can be successfully built.

- 1c9e029 and all versions of it so far reported some errors.

`cmake ..` works fine on both of them.
But when I am executing `MSBuild edge.vcxproj /t:Build /p:Configuration=Release` on 1c9e029 and all commits of it so far, These are some errors:

```
"F:\Workspace\Repository\repo\n2n\build\edge.vcxproj" (Build target) (1) ->
"F:\Workspace\Repository\repo\n2n\build\n2n.vcxproj" (default target) (3) ->
(ClCompile target) ->
  F:\Workspace\Repository\repo\n2n\edge_utils.c(580,12): error C2039:  “multicast_joined”: Not a member of "n2n_edge" [F:\Workspace\Repository\repo\n2n\build\n2n.vcxproj]
  F:\Workspace\Repository\repo\n2n\edge_utils.c(585,24): error C2039:  “udp_multicast_sock”: Not a member of "n2n_edge" [F:\Workspace\Repository\repo\n2n\build\n2n.vcxproj]
  F:\Workspace\Repository\repo\n2n\edge_utils.c(585,102): error C2198:  “setsockopt”: Too few parameters for calling [F:\Workspace\Repository\repo\n2n\build\n2n.vcxproj]
  F:\Workspace\Repository\repo\n2n\edge_utils.c(595,12): error C2039:  “multicast_joined”: Not a member of "n2n_edge" [F:\Workspace\Repository\repo\n2n\build\n2n.vcxproj]

    41 Warning(s)
    4 Error(s)
```

_(Prompt message has been translated)_

I built with:

- Windows 10 1903

- Visual Studio 2019

- The original repositories, only changed
`OPTION (N2N_OPTION_AES "USE AES" OFF)`
to
`OPTION (N2N_OPTION_AES "USE AES" ON)`
in CMakeLists.txt

I am in trouble now. I can't use the latest version. Please fix it!

Tests show that I can successfully build the latest version without any modifications on Ubuntu.

**Comment by emanuele-f** [comment]
Same error as #162, please check that issue for the follow up. Thank you for reporting

**Comment by lennondotw** [comment]
Fixed on 8880330!
